### PR TITLE
fix(console): split password from contact information, initialization mail on top

### PR DESCRIPTION
**Pull request by peintnermax**
Splitting password from contact information makes more sense

Bildschirmfoto 2022-09-15 um 16 44 55

after discussing with @mffap it would make sense to move password together with passwordless and mfa under one submenu.. what are your thoughts @juergrinaldi  ?

submenus would then be general -> rename to personal information?, security (password, mfa, passwordless), authorizations, memberships and metadata

### CODE REVIEW

**Comment by peintnermax** [comment]
Also moved initialization mail to the top

Bildschirmfoto 2022-10-03 um 14 17 03

**Review by eliobischof — COMMENTED** [review]
> Also moved initialization mail to the top
> 
> Bildschirmfoto 2022-10-03 um 14 17 03

Shouldn't this go to the contact information, as it is about the users **email** verification?

File: console/src/assets/i18n/en.json [hunk 1]
```diff
@@ -482,6 +482,7 @@
     "EMAILVERIFIED": "Email verified",
     "NOTVERIFIED": "not verified",
     "PREFERRED_LOGINNAME": "Preferred Loginname",
+    "ISINITIAL": "This user has not been verified yet. Click the button to send a new initialization mail.",
```
Comment (eliobischof) [inline console/src/assets/i18n/en.json:485]:
```suggestion
    "ISINITIAL": "This users email is not verified yet. Select the button to send a new initialization mail.",
```

File: console/src/assets/i18n/fr.json [hunk 2]
```diff
@@ -482,6 +482,7 @@
     "EMAILVERIFIED": "Courriel vérifié",
     "NOTVERIFIED": "non vérifié",
     "PREFERRED_LOGINNAME": "Nom de connexion préféré",
+    "ISINITIAL": "Cet utilisateur n'a pas encore été vérifié. Cliquez sur le bouton pour envoyer un nouveau mail d'initialisation.",
```
Comment (eliobischof) [inline console/src/assets/i18n/fr.json:485]:
```suggestion
    "ISINITIAL": "Le mail de cet utilisateur n'a pas encore été vérifié. Cliquez sur le bouton pour envoyer un nouveau mail d'initialisation.",
```

File: console/src/assets/i18n/it.json [hunk 3]
```diff
@@ -482,6 +482,7 @@
     "EMAILVERIFIED": "Email verificata",
     "NOTVERIFIED": "non verificato",
     "PREFERRED_LOGINNAME": "Loginname preferito",
+    "ISINITIAL": "Questo utente non è stato ancora verificato. Premi il pulsante per mandare una nuova e-mail di inizializzazione.",
```
Comment (eliobischof) [inline console/src/assets/i18n/it.json:485]:
```suggestion
    "ISINITIAL": "La e-mail di queste utente non è stato ancora verificato. Premi il pulsante per mandare una nuova e-mail di inizializzazione.",
```

File: console/src/assets/i18n/zh.json [hunk 4]
```diff
@@ -482,6 +482,7 @@
     "EMAILVERIFIED": "电子邮件已验证",
     "NOTVERIFIED": "未验证",
     "PREFERRED_LOGINNAME": "首选登录名",
+    "ISINITIAL": "该用户尚未通过验证。单击按钮触发新的初始化邮件。",
```
Comment (eliobischof) [inline console/src/assets/i18n/zh.json:485]: ? :-)

**Review by eliobischof — CHANGES_REQUESTED** [review]

File: console/src/assets/i18n/en.json (hunk 1 above)
Comment (eliobischof) [reply console/src/assets/i18n/en.json:485]: That's what @MattDodsonEnglish used when he reviewed our docs :-)
Maybe on mobile phones, you don't "click" buttons?

Comment (MattDodsonEnglish) [reply console/src/assets/i18n/en.json]: Exactly :-) . Even on Desktops though: some people prefer to navigate (or must navigate) applications with keyboard.
I stole this from Microsoft: https://github.com/zitadel/zitadel/pull/4380#discussion_r991450499

In the long run, I can't say it's a very important point of technical communication. It assumes a little less about the reader, though :slightly_smiling_face: .

Comment (MattDodsonEnglish) [reply console/src/assets/i18n/en.json]: But, the best strategy might be to just use the verb that the button represents; in this case it looks like "Send registration link". Generally, avoiding direct references to UI elements makes a doc more future-proof.

Comment (eliobischof) [reply console/src/assets/i18n/en.json]: > But, the best strategy might be to just use the verb that the button represents; in this case it looks like "Send registration link". Generally, avoiding direct references to UI elements makes a doc more future-proof.

I like this approach. Thanks for clarification :blush:

Comment (eliobischof) [reply console/src/assets/i18n/en.json]: @juergrinaldi do you also have an opinion on that?

**Comment by peintnermax** [comment]
> > Also moved initialization mail to the top
> > Bildschirmfoto 2022-10-03 um 14 17 03
> 
> Shouldn't this go to the contact information, as it is about the users **email** verification?

the section is shown on every page

**Review by eliobischof — APPROVED** [review]

**Pull request merged**